cli/tests/package-show.bats

```diff
@@ -250,7 +286,8 @@ teardown() {
   pushd 2
   "$FLOX_BIN" init
   _PKGDB_GA_REGISTRY_REF_OR_REV="${PKGDB_NIXPKGS_REV_NEW?}" \
-    "$FLOX_BIN" install nodejs
+    "$FLOX_BIN" update --global
+  "$FLOX_BIN" install nodejs
```

Comment:
suggestion: separate these commands
otherwise looks like a missed `\`

Also maybe add a comments why you call them with different envs

```suggestion
    "$FLOX_BIN" update --global
    
  "$FLOX_BIN" install nodejs
```

Reply:
Fixing